libs/sdk-ui-dashboard/src/model/commands/index.ts

```diff
@@ -153,7 +153,7 @@ export {
     ChangeInsightWidgetInsight,
     changeInsightWidgetInsight,
     ModifyDrillsForInsightWidget,
-    modifyDrillForInsightWidget,
+    modifyDrillsForInsightWidget as modifyDrillForInsightWidget,
```

Comment:
Incomplete renaming, please propagate the new name everywhere.

Reply:
Done